### ISSUE TITLE
A summary of information to the CLI runner would be useful.

### ISSUE THREAD

**Issue by alexsalesdev** [MutabilityDetector/MutabilityDetector#66]
As a user, I was expecting summary of information after running `java -jar Mutability-Detector.jar -cp <my_workspace>`

By having a commandline flag like -summary/-s i.e `java -jar Mutability-Detector.jar -summary -cp <my_workspace>`

at the end of the execution of CLI runner there would be a summary of information i.e

```
50 Total number of classes scanned.
34 IMMUTABLE classes.
16 NOT_IMMUTABLE classes.
0.0163257 seconds runtime.
```

**Comment by Grundlefleck** [comment]
Excellent idea. I like the suggested output as is.

I know from other channels you intend to contribute the change yourself, so look forward to the pull request! :smile:

**Comment by jagan387** [comment]
Isn't Summary already available? Is this also a fixed issue?

**Comment by Grundlefleck** [comment]
I have left this as open, but with the due-for-release tag, because there is no version available from Maven Central which contains the fix. There is no more work to be done on this issue except to cut a release. I'm unsure when exactly that will happen. 

I kinda wish there was another state for this issue, because I see how it's confusing for anyone looking for something to contribute, but I also think it would be confusing to see this issue as fixed but there being no version which includes it. @jagan387 do you think there's a better way I should manage this issue?

**Comment by jagan387** [comment]
I'm unsure about open source project conventions, but in enterprise, everything thats fixed in code/text is marked text. The release version is usually tracked by either a separate "release by" field, or having a "next-release" or "due-for-release" tags. 

Later, when we are about to release, we search the BugDB with either the release by field for the current version or the next-release tags to get all the bugs to be present in next release. Not really sure if this applies at all to this project

**Comment by jagan387** [comment]
Does the above idea make sense to any of you?

Even if it doesn't, we probably need to have a better way to differentiate issues which needs to be merged in the next release, and those which are open and needs to be fixed

It is really confusing to look at open issues, and read them all the way, to only find out that they have been fixed and merged

Can we please have a discussion to clean this up?

**Comment by Grundlefleck** [comment]
@jagan387 I've now closed this issue because 0.9.6 has been released.

I plan to do something different for tracking which issues need further work vs. those which are complete but not yet released. From now on, if an issue is open, that means there is further work to be done, and contributions are gratefully received.

Sorry for such a long delay in responding.

**Comment by jagan387** [comment]
@Grundlefleck Thanks for the decision of keeping only dev-work-needed issues as open. I will check back in a week, to try and pick up something that i can work on. Hopefully, by then, you will have enough time to clean up the existing open issues according to your future plan
